cms/templates/cms/program_page.html

```diff
@@ -1,12 +1,234 @@
 {% extends "base.html" %}
 
+{% load i18n static %}
 {% load wagtailcore_tags %}
 
+{% block title %}{% trans "MIT Micromasters" %}{% endblock %}
+
 {% block body_class %}template-programpage{% endblock %}
 
 {% block content %}
-<h1>{{ page.title }}</h1>
+  <div class="app-media layout-boxed">
+    <div class="main-content-wrapper">
+      <section class="main-content">
+        <div class="banner-wrapper banner-sub banner-program-1">
+          <nav class="navbar navbar-default" role="navigation">
+            <div class="container-fluid">
+              <div class="navbar-header">
+                <a class="navbar-brand" href="javascript:void(0)"><img
+                        src="/static/images/logo-mit-micromasters-whitered@2x.png"
```

Comment:
Can you change this to `logo-mit-micromasters-whitered.png` from the templates repository, then change width and height to be half their values?